pages/donate/index.tsx

```diff
@@ -7,12 +7,16 @@ const Donate: React.FC = () => {
     <>
       <PopUp
         header="Open different page"
-        body="This donation button will take you to MAC’s donation page where you will find a YELLOW button that says OUTstandingLIFE. All donations go through their secure account and are then credited to OUTstandingLIFE."
         manageModal={modalManagement}
         onConfirm={() => {
           window.open('https://www.mac-boston.org/donate', '_blank');
-        }}
-      />
+        }}>
+        <Text>
+          This donation button will take you to MAC’s donation page where you will find a YELLOW
```

Comment:
💭 Would it be good to show images or a gif displaying exactly what to press rather than describing the UI via text?

Reply:
This is the text that they gave us lol